chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -593,16 +601,40 @@ impl snowbridge_pallet_inbound_queue::Config for Runtime {
     type MessageProcessor = (benchmark_helper::WorstCaseMessageProcessor<EthTokensProcessor>,);
 }
 
+pub type RawMessageProcessorInboundV2 = RawMessageProcessorV2<
```

Comment:
what does raw mean? the plain snowbridge v2 processor? if so comment

Reply:
This is the processor for processing `RawPayload::Xcm` messages. 
Another one for processing `RawPayload::Symbiotic` messages.

Reply:
Added comment: https://github.com/moondance-labs/tanssi/pull/1420/commits/3ed5c189faa337fecba639fae1f3b495969cb7d8